public/plugin.tsx

```diff
@@ -74,7 +75,7 @@ export class AssistantPlugin
     const checkAccess = (account: Awaited<ReturnType<typeof getAccount>>) =>
       account.data.roles.some((role) => ['all_access', 'assistant_user'].includes(role));
 
-    if (this.config.chat.enabled) {
+    if (this.config.chat.enabled && this.config.chat.rootAgentId) {
```

Comment:
maybe we still want to show chat UI since it will receive `AgentIdNotFoundError` as response
https://github.com/opensearch-project/dashboards-assistant/blob/7862f42e7f518cc62a3ed0008d6902d55e3c1f93/server/routes/chat_routes.ts#L146-L149
to let web users know why chat is not available?

Reply:
Yeah, but the final solution has changed and no matter how we are gonna to remove the rootAgentId in request payload. Will optimize this part in another PR.